java/runtime/src/main/java/io/ray/runtime/AbstractRayRuntime.java

```diff
@@ -175,13 +177,10 @@ public PlacementGroup createPlacementGroup(
       throw new IllegalArgumentException(
           "Bundles cannot be empty or bundle's resource must be positive.");
     }
-    return taskSubmitter.createPlacementGroup(name, bundles, strategy);
-  }
-
-  @Override
-  public PlacementGroup createPlacementGroup(
-      List<Map<String, Double>> bundles, PlacementStrategy strategy) {
-    return createPlacementGroup(DEFAULT_PLACEMENT_GROUP_NAME, bundles, strategy);
+    Preconditions.checkNotNull(
```

Comment:
Judge whether strategy is null when creating creationOptions?

Reply:
yeah, I will move the bundles check to creationOptions too.